.github/workflows/quality.yml

```
@@ -99,32 +31,63 @@ jobs:
           --health-interval 10s
           --health-timeout 5s
           --health-retries 6
-
+    strategy:
+      fail-fast: false
+      matrix:
+        repositories:
+          [
+            "@socialgouv/cdtn-ui",
+            "@socialgouv/react-fiche-service-public",
+            "@cdt/frontend",
+            "@cdt/data",
+            "@socialgouv/modeles-social",
+            "@cdt/api",
+          ]
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
       - name: Get yarn cache directory path
         id: yarn-cache-dir-path
         run: echo "::set-output name=dir::$(yarn cache dir)"
-
-      - uses: actions/cache@v2
+      - name: Use cache
```

```suggestion
      - name: Restore cache
```
J'aurais plutôt dit que l'on restore le cache ici.